### DISCUSSION TITLE
The latest versions are not in opam

### DISCUSSION THREAD

**Discussion by vitoyucepi** [savonet/liquidsoap#2533] [General]
Just wonder if it's just me or everyone.

Liquidsoap `2.0.7`, `2.1.0` and `2.1.0-1` are not present in opam.
Also I can't find all new versions for packages since https://github.com/ocaml/opam-repository/commit/1a50221d.
I'm pretty sure it's problem on opam-repository side.
@toots could you check please.

**Comment by toots** [comment]
Seems to be working fine here. Have you checked if you had any pinned packages? `opam pin list`

**Reply by vitoyucepi** [reply]
I'm actually trying to use https://hub.docker.com/r/ocaml/opam.
So the solution is `opam repository set-url default https://opam.ocaml.org/`.

There is a copy of opam-repository repo inside the image, thus cache is fully local.
I didn't find clear instruction on how to switch the repo.
